tool/dev.dart

```diff
@@ -22,7 +22,7 @@ main(args) async {
   config.copyLicense.directories = directories;
   config.coverage.reportOn = ['bin/', 'lib/'];
   config.format
-    ..directories = directories
+    ..paths = directories
```

Comment:
one of the integration test fixtures should use the `paths` configuration as well, just to verify that code path